lib/ui/painting.dart

```diff
@@ -1568,6 +1568,17 @@ class Paint {
   }
 }
 
+/// The color space that an [Image] uses.
+enum ColorSpace {
+  /// The sRGB color gamut.
+  sRGB,
+  /// A color space that is backwards compatible with sRGB but can represent
+  /// colors outside of that gamut with values outside of [0..1]. In order to
+  /// see the extended values a [ImageByteFormat] like
```

Comment:
Mention where the [ImageByteFormat] can be used, since here we're looking at the enum docs and they don't link to anything.

Reply:
done, added link to `toByteData` to ColorSpace